metadata-db-app/test/cmr/metadata_db/test/data/oracle/granule_table.clj

```diff
@@ -0,0 +1,92 @@
+(ns cmr.metadata-db.test.data.oracle.granule-table
+  (:require
+  [clojure.test :refer :all]
```

Comment:
The require namespaces need to be moved over 1 space

`
(:require
 [clojure.test :refer :all]
`

Reply:
fixed